atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/TableDefinition.java

```diff
@@ -86,8 +87,11 @@ public void namedComponentsSafeByDefault() {
 
     /**
      * If specified, makes both the table name and ALL named components safe by default.
-     * The table name may still be marked as unsafe by setting tableNameLogSafety(), and individual row components
-     * or named columns may still be marked as unsafe by explicitly creating them as unsafe.
+     * Individual row components or named columns may still be marked as unsafe by explicitly creating them as unsafe.
+     *
+     * This method has UNDEFINED BEHAVIOUR if used with tableNameLogSafety(LogSafety.UNSAFE) on the same table.
```

Comment:
It's possible, of course. (Keep track both _if safety has been set_ and _what it has been set to, if applicable_). I didn't initially think it was worth the complexity, though might be better to fail-fast and throw an exception here. Will change this.